view-admin/src/components/common/JudgementModal/JudgementModal.module.css

```
@@ -121,42 +124,41 @@
 
 .judgementContents {
   display: flex;
+  justify-content: center;
 }
 
 .jugementButton {
```

```suggestion
.judgementButton {
```
前からタイポしてたみたい
modal.tsxもstyles.judgementButtonに修正して